kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -145,30 +144,25 @@
         // creates array of words in filter, removes empty strings
         const searchFilter = this.searchFilter.split(' ').filter(Boolean).map(
           // returns an array of search parameters, ignoring case
-          (query) => new RegExp(query, 'i'));
+          (query) => {
+            const escapedQuery = query.replace(/[-\/\\^$*+?.()|[\]{}]/g, '\\$&');
```

Comment:
This is very confusing and potentially brittle. I can't think of a good reason to be using regex here, especially 'auto-generated' regex patterns based on user input.

A more simple and robust way to do case insensitive string matching is to just convert both strings to upper case and then search for the substring.